akinet/http/parser_factory_test.go

```diff
@@ -226,6 +226,18 @@ func TestHTTPResponseParserFactoryAccepts(t *testing.T) {
 			expectedDecision: akinet.Accept,
 			expectedDF:       int64(len("OK")),
 		},
+		// This is technically illegal but we observed some Postman services sending it.
+		{
+			name:             "no reason message",
+			input:            "HTTP/1.1 200\r\n",
+			expectedDecision: akinet.Accept,
+		},
+		// This is technically legal, as the reason string can be zero bytes long.
```

Comment:
Or we can just right `This is technically legal as reason string is optional` for better readability.

Reply:
I think it is less clear what "optional" means than the specific thing this test is demonstrating